Makefile

```diff
@@ -51,13 +51,13 @@ MINIKUBE_RELEASES_URL=https://github.com/kubernetes/minikube/releases/download
 
 KERNEL_VERSION ?= 4.15
 # latest from https://github.com/golangci/golangci-lint/releases
-GOLINT_VERSION ?= v1.18.0
+GOLINT_VERSION ?= v1.20.0
 # Limit number of default jobs, to avoid the CI builds running out of memory
 GOLINT_JOBS ?= 4
 # see https://github.com/golangci/golangci-lint#memory-usage-of-golangci-lint
-GOLINT_GOGC ?= 8
+GOLINT_GOGC ?= 100
```

Comment:
what is the reason for increasing 8 to 100 ?

Reply:
GOGC=100 is the default, so it is basically removing the increased garbage collection

Reply:
Yes, I restored the default, but I left the link to the docs in case that setting needs to be adjusted in the future.